### PR TITLE
SDK-1295: Added condition to check ENV setting and config

### PR DESCRIPTION
**Pull request by vol4onok**
## Issue ticket link
Ticket: https://spryker.atlassian.net/browse/SDK-1295.

## Checklist before requesting a review
- [ ] I have added ticket link to the PR description.
- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by sergeyspryker — DISMISSED** [review]

File: bin/spryker-sdk.sh [hunk 1]
```diff
@@ -119,6 +119,10 @@ case $MODE in
     if [[ -z "$SPRYKER_SDK_ENV" || $SPRYKER_SDK_ENV == 'prod' ]]; then
         docker-compose -f "${SDK_DIR}/docker-compose.yml" run --rm spryker-sdk "$ARGUMENTS"
     else
+        if [[ ! -f "${SDK_DIR}/docker-compose.dev.yml" ]]; then
+           echo "SPRYKER_SDK_ENV=$SPRYKER_SDK_ENV environment is not available for installer. Remove SPRYKER_SDK_ENV env variable if it exists or set it value to \"prod\" by running \"export SPRYKER_SDK_ENV=prod\""
```
Comment (dereuromark) [inline bin/spryker-sdk.sh:123]: Can we please use ` for code fencing. " is not proper for this.

Comment (dereuromark) [reply bin/spryker-sdk.sh]: This was not fixed now in the way it should be.

Comment (pavelmaksimov25) [reply bin/spryker-sdk.sh]: I'm not sure, but afaik ` can be recognized as a command. Example
```shell
[spryker@SDK_CLI]$ x="`pwd`"
[spryker@SDK_CLI]$ echo $x
/data
[spryker@
```

File: bin/spryker-sdk.sh [hunk 2]
```diff
@@ -111,6 +111,10 @@ case $MODE in
     if [[ -z "$SPRYKER_SDK_ENV" || $SPRYKER_SDK_ENV == 'prod' ]]; then
         docker-compose -f "${SDK_DIR}/docker-compose.yml" run --rm spryker-sdk "$ARGUMENTS"
     else
+        if [[ ! -f "${SDK_DIR}/docker-compose.dev.yml" ]]; then
```
Comment (pavelmaksimov25) [inline bin/spryker-sdk.sh:114]: elif

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]

File: bin/spryker-sdk.sh (hunk 1 above)
Comment (pavelmaksimov25) [reply bin/spryker-sdk.sh]: done

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**